src/components/header/Header.tsx

```diff
@@ -12,7 +12,10 @@ const Header: React.FC<HeaderProps> = ({ headerTitle }) => {
       <div className='BottomBorder--border heading--container'>
        <img src={mentalHealthLogo} alt="head outline with lotus flower" className="logo--landing-page"></img>
       {headerTitle !== 'HeLP Network' && <h1 className='HeaderTitle--h1'>{headerTitle}</h1>} 
-        {headerTitle === 'HeLP Network' && <h1 className='HeaderTitle--h1'>H<span className="letter-standout">e</span>LP Network</h1>}
+        {headerTitle === 'HeLP Network' && <>
+          <h1 className='HeaderTitle--h1'>H<span className="letter-standout">e</span>LP Network</h1>
+          <p className="subheading">Find answers to important legal and ethical questions</p>
+        </>}
```

Comment:
I love this. Simple but very effective. Great job getting and implementing such great feedback.